docs/proposals/registry/sample-vs-stack.md

```diff
@@ -0,0 +1,195 @@
+# Proposal for sample vs stack in a devfile registry
+
+## Overview
+
+The existing devfile registry focuses on the stack support to provide generic language/framework/runtime, e.g. Node, Java maven, Quarkus, etc.,  support to build and run user applications. These devfiles are called ___*stack devfiles*___.  There is another kind of devfile in a devfile registry that is tailored for a building and running a specific application. These devfiles are referred to as ___*sample devfiles*___ (example: https://github.com/redhat-developer/devfile-sample).
```

Comment:
I think e.g. `***sample devfiles***` looks more markdown-y and renders the same as `___*sample devfiles*___`